bitsandbytes/__main__.py

```diff
@@ -46,26 +32,31 @@ def generate_bug_report_information():
     print('')
 
     if 'CONDA_PREFIX' in os.environ:
-        paths = find_file_recursive(os.environ['CONDA_PREFIX'], '*cuda*so')
+        paths = find_file_recursive(os.environ['CONDA_PREFIX'], '*cuda*')
         print_header("ANACONDA CUDA PATHS")
         print(paths)
         print('')
     if isdir('/usr/local/'):
-        paths = find_file_recursive('/usr/local', '*cuda*so')
+        paths = find_file_recursive('/usr/local', '*cuda*')
         print_header("/usr/local CUDA PATHS")
         print(paths)
         print('')
+    if 'CUDA_PATH' in os.environ and isdir(os.environ['CUDA_PATH']):
+        paths = find_file_recursive(os.environ['CUDA_PATH'], '*cuda*')
```

Comment:
Same as above. Can we ensure correctness in weird envs?

Reply:
> Same as above. Can we ensure correctness in weird envs?

currently, `find_file_recursive()` is used for debugging purposes only (used at `generate_bug_report_information()`).
so I think it is enough.

Reply:
answer makes sense to me